about.html

```diff
@@ -0,0 +1,164 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Capstone</title>
+     <!-- CSS STYLE LINK -->
+     <link rel="stylesheet" href="style.css">
+     <link rel="stylesheet" href="about.css">
+     <!-- ICONTSCOUNTS LINK -->
+     <link rel="stylesheet" href="https://unicons.iconscout.com/release/v2.1.6/css/unicons.css">
+     <!-- GOOGLE FONTS LINKS -->
+     <link rel="preconnect" href="https://fonts.googleapis.com">
+     <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+     <link href="https://fonts.googleapis.com/css2?family=Montserrat:wght@300;400;500;600;700&display=swap" rel="stylesheet">
+ 
+</head>
+<body>
+  <div class="nav-container-bg">
+    <nav class="top-navbar">
+      <ul>
+        <li><img src="imgs/facebook.png" alt="facebook-image"></li>
+        <li><img src="imgs/twitter.png" alt="twitter-image"></li>
+        <li>logout</li>
+        <li>Mypage</li>
+        <li>Korean</li>
+      </ul>
+    </nav>
+    <nav>
+      <div class="container nav-container">
+        <div class="hamburger-btn">
+          <span class="bar"></span>
+          <span class="bar"></span>
+          <span class="bar"></span>
+        </div>
+        <div class="menu-js">
+          
+          <button id="close-btn" type="button" class="close-btn x">
+            <span>&times;</span>
+          </button>
+        <div class="image-nav">
+          <a href="index.html"> <img class="progarm-logo" src="imgs/logo-project.png" alt=""></a>
+          <ul class="nav-menu">
+              <li ><a class="link about"  href="about.html">About</a></li>
+              <li ><a class="link" href="">Program</a></li>
+              <li ><a class="link" href="">Join</a></li>
+              <li ><a class="link" href="">Sponser</a></li>
+              <li ><a class="link" href="">News</a></li>
+              <li ><a class="link cc-compagin" href="">CCCampagin</a></li>
+          </ul>
+        </div>
+         </div>
+      </div>
+    </nav>
+    <!-- END OF NAVBAR --> 
+
+    <!-- FIRST SECTION-->
+    <section class="container container-intro-about">
+        <div class="project-head-title-about">
+          <h3 class="project-title-about">"Hello! Sharing world"</h3>
+          <h2 class="project-desc-about">Creative commons global summit 2015</h2>
+        </div>
+        
```

Comment:
- [ ] Kindly edit this section's header text to have the appropriate size and spacing around it to match the original design.

| | Your design | Original design |
| --- | --- | --- |
| Desktop | ![image](https://user-images.githubusercontent.com/62290545/170581304-e29b38ae-de8f-4c68-be2c-297a3fc7c356.png) | ![image](https://user-images.githubusercontent.com/62290545/170581352-02ccdb6d-9f24-44cc-8028-6c6d707ee202.png) |
| Mobile | ![image](https://user-images.githubusercontent.com/62290545/170581406-ab761c7b-c3de-4b25-9db1-6fdb48874a98.png) | ![image](https://user-images.githubusercontent.com/62290545/170581455-42f829a7-d0a5-4ef2-b383-1ddc9ab08c2d.png) |